README.md

```diff
@@ -18,14 +18,16 @@ To run the REPL just enter the following in the source code directory
 
 and start typing REPL commands there. The syntax of the REPL commands is
 
-    <repl-command> := <term> | <binding>
+    <repl-command> := <term> | <binding> | <loading>
     <term> ::= <variable>
              | <numeric-literal>
              | <character-literal>
              | <string-literal>
              | <function>
              | <application>
     <binding> ::= <variable> := <term>
+    <loading> ::= load <module-path>
+    <module-path> := [a-zA-Z][a-zA-Z0-9/]*
```

Comment:
I believe it was meant as a filesystem paths because no sane module system includes `/` as a namespace separator.

_(PHP **does** and that's an exact reason it's not sane)_

Maybe replace `/` with `.` there? It will state your intention more clear.